src/public/warehouse.ts

```diff
@@ -26,8 +26,7 @@ export const itemPublicHandlers: PublicNatsHandler[] = [
                         "item-store.add_warehouse_item",
                         jsonCodec.encode(body)
                     );
-
-                    message.respond(response.data);
+                    message.respond(response.data)
```

Comment:
this is missing a semi colon `;`, could re-run `npm run prettier` and `npm run lint` to reformat and verify that there's no other linting issue?